components/formats-bsd/src/loci/formats/Memoizer.java

```diff
@@ -327,6 +327,17 @@ public void saveStop() throws IOException {
 
   }
 
+  private static Object getMetadataOption(
```

Comment:
I wonder if there's useful scope for making this a `private static <X> X get...(... Class<X> type)` or somesuch then having the logging instead catch the exception from `type.cast`. (Something like that anyway.)